extensions/1bookmark/README.md

```diff
@@ -0,0 +1,10 @@
+# 1Bookmark - Raycast Extension
```

Comment:
I'm missing some more about the service used and also how to use it (create user/login) and so on